src/signals/incident-management/containers/Dashboard/Dashboard.tsx

```diff
@@ -1,32 +1,37 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2023 Gemeente Amsterdam
-import { useCallback, useEffect, useState } from 'react'
-
-import GlobalError from 'components/GlobalError'
+// import { useCallback, useEffect, useState } from 'react'
+//
+// import GlobalError from 'components/GlobalError'
 
+import { Filter } from './components/Filter'
 import { StyledRow } from './styled'
 
 const Dashboard = () => {
-  const [error] = useState<boolean>(false)
-  const [showMessage, setShowMessage] = useState<boolean>(false)
-  const [errorMessage, setErrorMessage] = useState<JSX.Element | string>('')
+  // const [error] = useState<boolean>(false)
```

Comment:
these commented lines should be implemented when starting 

https://gemeente-amsterdam.atlassian.net/browse/SIG-4900?atlOrigin=eyJpIjoiNTAzZmYzNjU3MTcwNDFmZGI0ZmE4NTc0MjA2N2NlY2QiLCJwIjoiaiJ9

Reply:
oke, thought the same thing